src/internal/evaluation/EvaluationStorage.ts

```diff
@@ -0,0 +1,64 @@
+import { Evaluation } from '../model/Evaluation'
+import { BKTStorage } from '../storege'
+
+export interface EvaluationEntity {
+  userId: string
+  evaluations: Record<string/* featureId */, Evaluation>
+}
+
+export interface EvaluationStorage {
+  add( evaluations: Evaluation[]): void
+  get(): Record<string, Evaluation>
+  getByFeatureId(featureId: string): Evaluation | null
+  deleteAllAndInsert(evaluations: Evaluation[]): void
+}
+
+export class EvaluationStorageImpl implements EvaluationStorage {
```

Comment:
I'm planning to implement EvalutinInteractor as I did in Android SDK, but not going to add an in-memory cache and use this EvaluationStorage directly.

LocalStorage should be fast enough.

Reply:
Sounds good.
Do you know if the current SDK is using an in-memory cache?

Reply:
Yes, it's using in-memory cache.
https://github.com/ca-dp/bucketeer-web-sdk/blob/master/src/storage/storage.ts#L31